content/blog/convince-my-boss.md

```diff
@@ -0,0 +1,21 @@
+Dear \{bossName\},
+
+I am excited to brief you about PyCon India, India’s premier conference on using and developing the Python programming language that takes place in Chennai from October 12\-15, 2019\. About a 1000\+ of the industry's Pythonistas from different fields are bound to be in attendance for the 11th edition of this mega event\.
+
+It would be great if I get an opportunity to attend this Conference as there will be Innumerable benefits for me as an individual which will in turn impact the Organization as well\. To list a few:
+
+- Am Particularly keen on attending this <talk/workshop/devsprint> on <topics> presented by <speaker name> as our project is based on this <or upcoming project based on this>
+- Exposure to the Actionable Best Practices & tools that we could use in our projects, saving money and time spent exploring different options\.
+- Provides a forum to get a sneak peek at upcoming Innovations
+- Provides a platform to unmask the rationality behind the trending technologies like ML/AI, Automation, Web development, Software development, IOT and more that uses python
+- Learning Technologies from a wide array of Talks, Workshops, Keynotes speech, Developer Sprints each presented by inspiring and enthusiastic Industry experts, community leaders and many young minds delving into new technologies
+- Opportunity to network with like\-minded people from diverse backgrounds that includes Geek Programmers, Enthusiasts, Entrepreneurs, Scholars, Intellectual and many more
+- Platform to promote our Company’s brand by showing a thorough knowledge of our company, product/services we offer, create strategic partnership\. Also, if necessity we can grab our company future employees from this diversified talent pool\.
```

Comment:
How to promote company's brand is not clear here.

Reply:
+ A chance to spread the word about our Company and create goodwill about the opportunities and incentives the employees are given. This will the ideal way to attract future employees from an diversified talent pool.